src/components/CharacterForm/CharacterForm.js

```diff
@@ -132,13 +137,17 @@ const CharacterForm = ({ setOpen }) => {
     },
   }
 
+  if (!params) return
```

Comment:
You shouldn't be doing this, instead you should be validating the URL to make sure it is as expected and responding accordingly when it's not. This should be done in the main page

Reply:
I have made changes that makes it work as in the error that causing for me before will not appear anymore. But I'm not really sure on making url validation

Reply:
As we discussed, you can do URL validation inside a useEffect